.github/libs/GithubUtils.js

```diff
@@ -273,11 +276,12 @@ class GithubUtils {
                     });
                 }
 
+                // Internal QA PR list
                 if (!_.isEmpty(internalQAPRMap)) {
                     issueBody += '\r\n\r\n\r\n**Internal QA:**';
                     _.each(internalQAPRMap, (assignees, URL) => {
                         const assigneeMentions = _.reduce(assignees, (memo, assignee) => `${memo} @${assignee}`, '');
-                        issueBody += `\r\n${_.contains(verifiedOrNoQAPRs, URL) ? '- [x]' : '- [ ]'} `;
+                        issueBody += `\r\n${_.contains(resolvedInternalQAPRs, URL) ? '- [x]' : '- [ ]'} `;
```

Comment:
And finally check if the URL is in the `resolvedInternalQAPRs` array as the `verifiedOrNoQAPRs` does not include the necessary InternalQA PRs with `isResolved` field, hence it was always false and it have reset the checkbox to unchecked state.